### PR TITLE
TGP-1392: Update Profile Journey

### PR DESCRIPTION
**Pull request by besscerule**

### CODE REVIEW

File: app/controllers/UkimsNumberController.scala [hunk 1]
```diff
@@ -47,27 +54,71 @@ class UkimsNumberController @Inject() (
...
+              }
+          }
+      )
+  }
+
+  def logErrorsAndContinue(errors: data.NonEmptyChain[ValidationError]): Result = {
+    val errorMessages = errors.toChain.toList.map(_.message).mkString(", ")
+
+    // TODO: got to profile page
+    val continueUrl = RedirectUrl(routes.HomePageController.onPageLoad().url)
```
Comment (Ewan-James-Donovan) [inline app/controllers/UkimsNumberController.scala]: Why do we send to homepage if something goes wrong? I'd expect journery recovery, or is there a reason for this being the home page? There seems to be a todo also

Comment (besscerule) [reply app/controllers/UkimsNumberController.scala]: i'm waiting for Yohan's profile page - hold your horses :P

Comment (Ewan-James-Donovan) [reply app/controllers/UkimsNumberController.scala]: holding :)

Comment (Ewan-James-Donovan) [reply app/controllers/UkimsNumberController.scala:118]: holding :)

**Review by Ewan-James-Donovan — DISMISSED** [review]

**Review by AmarkumarJagtap — DISMISSED** [review]
looks good to me

**Review by liandra-giacomine — COMMENTED** [review]
As discussed, this looks good just needs a renaming of the HasNirmsChange controller to RemoveNirms :)

**Review by liandra-giacomine — APPROVED** [review]

**Pull request merged**